metis-enrichment/metis-enrichment-common/src/main/java/eu/europeana/enrichment/api/external/model/EnrichmentQuery.java

```diff
@@ -0,0 +1,76 @@
+package eu.europeana.enrichment.api.external.model;
+
+// Request class for sending request to Entity API client
+public class EnrichmentQuery {
```

Comment:
Is there a particular reason for this class?
We have a whole set of objects that represent the search.
EnrichmentReference
EnrichmentSearch
ReferenceValue
SearchValue

Reply:
Ok I'll see what I can do.